src/components/React/Controls.js

```diff
@@ -17,31 +15,13 @@ import {
 
 import OpacityControl from "./OpacityControl";
 import ColorMapControl from "./ColorMappingController";
-import { Checkbox } from "primereact/checkbox";
-import { Dropdown } from "primereact/dropdown";
-
-// const options = [
-//   { value: './assets/models/48hr_20x_23_0.597976_ 0.597976_5.png:false', label: 'Spheroid old' },
-//   { value: './assets/models/r06c03f04_49_1.29_1.29_5.00000.png:false', label: 'Spheroid 1 cropped slices' },
-//   { value: './assets/models/r06c03f04_90_1.935_1.935_5.00000.png:false', label: 'Spheroid 1 90 slices' },
-//   { value: './assets/models/r06c03f03_49_1.29_1.29_5.00000.png:false', label: 'Spheroid 2 cropped slices' },
-//   { value: './assets/models/r06c03f03_90_1.935_1.935_5.00000.png:false', label: 'Spheroid 2 90 slices' },
-// ];
-
-const channelOptions = [
-  { value: 6, label: "Default" },
-  { value: 1, label: "Red" },
-  { value: 2, label: "Green" },
-  { value: 3, label: "Blue" },
-  { value: 4, label: "Alpha" },
-];
+import { Container, Form, Row } from "react-bootstrap";
 
 const Range = Slider.Range;
 
 const mapStateToProps = (state) => {
   return {
```

Comment:
what's the effort level to transition this to a functional component?

Reply:
Can be done when transitioning to context from redux - #40